### ISSUE TITLE
The PDF export function is not friendly

### ISSUE THREAD

**Issue by xuyikai1** [BoostIO/Boostnote#3576]
I export my notes to PDF format. If a line is too long, there will be a lot of content in the exported PDF line that will not be displayed

source：
![image](https://user-images.githubusercontent.com/26078579/82282455-798bc980-99c6-11ea-962c-6f3db884be99.png)

PDF：
![image](https://user-images.githubusercontent.com/26078579/82282378-477a6780-99c6-11ea-9cfe-8776930d5c98.png)

As shown in the figure, the bar below the code can't be pulled to the right, so you can't view the code that is blocked on the right

May I ask if this function can be optimized

version Boostnote 0.15.3

**Comment by Flexo013** [comment]
Duplicate of #2776
